main/settings.py

```diff
@@ -28,9 +29,9 @@
 SECRET_KEY = "7ibm!g0j@gs7kszwav!6$*lar(+!!l3tpm@09s%csl2bj)l+p4"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get("DJANGO_DEBUG", "") != "False"
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["https://open-source-programs-backend.herokuapp.com"]
```

Comment:
No need to add `https` in allowed host
```suggestion
ALLOWED_HOSTS = ["open-source-programs-backend.herokuapp.com"]
```

Reply:
Better to change the name to `anitab-forms-backend`